### PR TITLE
Sparkideer

### PR DESCRIPTION
**Pull request by charizard8888**
<h1>Ready to be merged </h1>

### CODE REVIEW

**Comment by charizard8888** [comment]
@XpRienzo @Spandamn merge and hotpatch! Thanks

**Comment by Spandamn** [comment]
I am dieing to know, did you code this?

File: mods/thefirstnewgen/pokedex.js [hunk 1]
```diff
@@ -498,4 +498,26 @@ exports.BattlePokedex = {
...
+	/* Mega Snorlax */
+	snorlax: {
+		inherit: true,
+		abilities: {0: "Immunity", 1: "Thick Fat", H: "Gluttony"},
+		otherFormes: ["snorlaxmega"],
+	},
+	snorlaxmega: {
+		num: 143,
+		species: "Snorlax-Mega",
+		baseSpecies: "Butterfree",
```
Comment (Spandamn) [inline mods/thefirstnewgen/pokedex.js]: Lol wut

**Comment by charizard8888** [comment]
Ah Dem dat Butterfree

**Comment by charizard8888** [comment]
Change that to Snorlax my bad :(

**Comment by Spandamn** [comment]
Please commit to do it
And you didn't answer, who coded this?

**Comment by charizard8888** [comment]
I coded it!

**Comment by charizard8888** [comment]
BOi dat thing is awesome!
/me praises @Spandamn for coading /distor in such a short period of time.... a command which we thought was Mission Impossible 8888!
We need that for Pokemon The New First Gen as well tbh

**Pull request merged**